File: python/tests/test_934.py
from python.src.code_934 import Solution
from python.tests.python_deps.boilerplate import BoilerTest


def input_fetcher(case):
    return case["input"]["grid"]


def test():
    test = BoilerTest(934, input_fetcher)
    solution = Solution().shortestBridge
    test.run_tests(solution)

File: cpp/test/test_934.cpp
#include "../src/code_934.cpp"
#include "cpp_deps/boilerplate.hpp"

void test(Solution& sol, const json& input, const json& output) {
    vector<vector<int>> grid = input["grid"].get<vector<vector<int>>>();
    int expected = output.get<int>();
    int result = sol.shortestBridge(grid);
    CHECK_EQ(result, expected);
}

// asdf
TEST_CASE("") {
    TEST(934);
}